src/index.css

```diff
@@ -0,0 +1,27 @@
+body {
```

Comment:
- [ ] OPTIONAL
The general layout of some of your pages **do not match** the provided [wireframe](https://github.com/microverseinc/curriculum-react-redux/tree/main/math-magicians/images). Kindly ensure to improve upon the following:
- [ ] The Home page should have some **paragraph text** after the heading _Welcome to our page_.
- [ ] The text content on the Quote page should be **vertically centered**.

**Your Pages:**
![image](https://user-images.githubusercontent.com/12745474/141499940-f5e3e7a0-7b0d-40ec-9a41-575147734267.png)

![image](https://user-images.githubusercontent.com/12745474/141499993-220515fe-d027-4609-8026-a7a20287883f.png)

**The wireframe pages:**
![image](https://user-images.githubusercontent.com/12745474/141191446-6a4d7184-1fb6-4a29-9423-29d2d0236649.png)

![image](https://user-images.githubusercontent.com/12745474/141191558-6c4e3c81-046b-4364-b097-3b2482d435b4.png)